duckduckgo/macos-browser
# Show preferences instead of Bitwarden app in case of communication error

Task/Issue URL: https://app.asana.com/0/1177771139624306/1204302365133777/f
CC:

**Description**:
If autofill message says "Bitwarden is locked" but communication is broken, it's more useful to navigate to our Autofill Settings instead of Bitwarden app. The Settings page contains an error message and provides an option to go through the onboarding process again.

**Steps to test this PR**:
1. Go to Settings -> Autofill and set Bitwarden as password manager
2. Finish the onboarding flow and make sure the autofill of Bitwarden passwords work
3. Quit Bitwarden app
4. Try to use autofill again and make sure "Bitwarden is locked" autofill message opens Bitwarden after clicking on it

5. Quit DuckDuckGo app
6. Open Keychain and remove `com.duckduckgo.macos.browser.debug.bitwarden.shared.key `
7. Open DuckDuckGo app again
8. Try to use autofill again but this time "Bitwarden is locked" autofill message must open Settings -> Autofill
9. Make sure the Settings page contains error message



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
